.github/workflows/build.yml

```diff
@@ -32,6 +32,7 @@ jobs:
     - name: Install dependencies
       run: |
         pip install -r requirements.txt
+        pip install -r requirements_dev.txt
```

Comment:
plz add a new section. Use black to check the code.

Reply:
working rn, btw I'm not so sure why one of the tests failed, seems that is an internal error in PyPy3?